src/components/CustomerQuota/CustomerQuotaScreen.tsx

```diff
@@ -256,10 +264,22 @@ export const CustomerQuotaScreen: FunctionComponent<CustomerQuotaProps> = ({
     clearError,
     error,
     expireSession,
-    showAlert
+    showAlert,
+    quotaState,
+    quotaError
   ]);
 
-  return cartState === "FETCHING_QUOTA" ? (
+  useEffect(() => {
+    /**
+     * Update quota after checkout
+     */
+    if (cartState === "PURCHASED" && !updateAfterPurchased) {
+      updateQuota();
+      setUpdateAfterPurchased(true);
+    }
+  }, [cartState, updateQuota, updateAfterPurchased]);
```

Comment:
Looks good, we only need to update the quota once after purchase after all 👍

Reply:
yeah! i realized it gets stuck in an update loop so this was necessary to get out of it!